frontend/pages/dashboard/open-call/[openCallId]/application/[applicationId].tsx

```diff
@@ -0,0 +1,235 @@
+
+import { getEnums } from 'services/enums/enumService';
+import { useOpenCallApplication } from 'services/open-call/application-service';
+
+const SECTION_CLASSNAMES = 'break-all';
+const GRID_CLASSNAMES = 'grid grid-cols-1 lg:grid-cols-[240px_1fr] gap-y-4';
+const LABEL_CLASSNAMES = 'flex items-center text-sm font-semibold text-gray-800';
+const LINK_CLASSNAMES =
+  'inline-flex gap-2 px-2 -mx-2 underline rounded-full text-green-dark focus-visible:outline focus-visible:outline-green-dark focus-visible:outline-2 focus-visible:outline-offset-2';
+
+export const getServerSideProps = withLocalizedRequests(async ({ locale }) => {
+  const enums = await getEnums();
+
+  return {
+    props: {
+      intlMessages: await loadI18nMessages({ locale }),
+      enums: groupBy(enums, 'type'),
+    },
+  };
+});
+
+type OpenCallDetailsPageProps = {
+  enums: GroupedEnumsType;
+};
+
+export const OpenCallDetailsPage: PageComponent<OpenCallDetailsPageProps, DashboardLayoutProps> = ({
+  enums,
+}) => {
+  const placeholderPicture = '/images/placeholders/profile-logo.png';
+
+  const [projectDeveloperPhoto, setProjectDeveloperPhoto] = useState<string>(placeholderPicture);
+
+  const intl = useIntl();
+  const router = useRouter();
+
+  const {
+    data: openCallApplication,
+    isLoading,
+    isLoadingError,
+  } = useOpenCallApplication(router.query.applicationId as string, {
+    includes: ['open_call', 'project', 'project_developer'],
+  });
+
+  const {
+    open_call: openCall,
+    project_developer: projectDeveloper,
+    project,
+  } = openCallApplication || {};
+
+  useEffect(() => {
+    if (!projectDeveloper?.picture?.small) return;
+    setProjectDeveloperPhoto(projectDeveloper?.picture?.small);
+  }, [projectDeveloper]);
+
+  const breadcrumbsProps = {
+    substitutions: {
+      'open-call': {
+        name: intl.formatMessage({
+          defaultMessage: 'Open calls',
+          id: 'OBhULP',
+        }),
+        link: Paths.DashboardOpenCalls,
+      },
+      openCallId: { name: openCall?.name },
+      applicationId: { name: project?.name },
+    },
+    hidden: ['dashboard', 'application'],
+  };
+
+  const projectDeveloperTypeName = useMemo(
+    () =>
+      enums[EnumTypes.ProjectDeveloperType].find(
+        ({ id }) => id === projectDeveloper?.project_developer_type
+      )?.name,
+    [enums, projectDeveloper]
+  );
+
+  // If we can't load the open call, it may have been removed or the user not have access to it. Let's
+  // redirect the user to the Dashboard open calls list.
+  if (isLoadingError) {
+    router.push(Paths.DashboardOpenCalls);
+    return null;
+  }
+
+  return (
+    <ProtectedPage permissions={[UserRoles.Investor]}>
+      <Head
+        title={`${
+          project?.name || intl.formatMessage({ defaultMessage: 'Project', id: 'k36uSw' })
+        } ${intl.formatMessage({
+          defaultMessage: 'application',
+          id: 'HFalmD',
+        })}`}
```

Comment:
I believe this should be one string only: `{projectName} application` and `projectName` is either `project?.name` or the translated string “Project”.

Reply:
You're right, makes more sense the way you described it.